mix.exs

```diff
@@ -54,7 +67,8 @@ defmodule Membrane.Element.Pcap.MixProject do
       {:membrane_core, "~> 0.2.0"},
       {:ex_doc, "~> 0.19", only: :dev, runtime: false},
       {:dialyxir, "~> 1.0.0-rc.4", only: [:dev], runtime: false},
-      {:expcap, github: "Hajto/expcap"}
+      {:expcap, github: "Hajto/expcap"},
```

Comment:
Remember we won't be able to publish it on Hex if it has a github dependency, so maybe create a PR for that guy

Reply:
We discussed it face to face